pyiron_base/state/settings.py

```diff
@@ -372,10 +379,48 @@ def _get_config_from_environment(self) -> Union[Dict, None]:
             elif k in self.environment_credential_map:
                 config[self.environment_credential_map[k]] = v
         config = self._fix_boolean_var_in_config(config=config)
-        config = self._add_credentials_from_file(config)
         return config if len(config) > 0 else None
 
-    def _add_credentials_from_file(self, config: dict) -> Dict:
+    def _add_credentials_from_file(self) -> Dict:
+        if (
```

Comment:
It might be nice to have an `else` clause that adds something to the log about no credentials file being found.

Reply:
This would only be something for a debug log imoh. This is just like that option has not been used in the main credentials. In that case we first should start to warn for unrecognized keys in the config such that users may ceck the spelling.